DATA/SConscript

```diff
@@ -205,10 +222,37 @@ bias = env.Command([os.path.join(REPO_ROOT, 'ci_cpp_bias'),
                     ])
 env.Alias('bias', bias)
 
-# Create DARK
+# Create DARK FOR DEFECTS
+dark_for_defects = env.Command([os.path.join(REPO_ROOT, 'ci_cpp_dark_for_defects'),
+                                os.path.join(REPO_ROOT, 'calib', 'v00', 'dark_for_defects')],
+                               bias,
+                               [getPipeTaskCmd('dark_for_defects', exposureDict['darkExposures'], 'cpDarkForDefects.yaml')])
+                                # getCertifyCmd('dark_for_defects'),])
```

Comment:
If we're not going to certify these ever, let's remove this comment (and the one for flat below).

Reply:
Good catch, these are dead lines.